/**:
  ros__parameters:
    parent_frame: base_link
    child_frame: event_camera
    x: 0.0
    y: 0.0
    z: 0.30
    roll_deg: 0.0
    pitch_deg: 40.0
    yaw_deg: 0.0
